qatrack/parts/views.py

```
@@ -308,7 +308,7 @@ def partstoragecollection__storage__room__name(self, p):
 
     def locations(self, obj):
         locs = self.parts_locations_cache.get(obj.id, {'locations': []})['locations']
-        return locs or _("None in storage")
+        return mark_safe(locs or _("None in storage"))
```

Ruff (0.15.14)

[error] Use of `mark_safe` may expose cross-site scripting vulnerabilities

(S308)